README.md

```diff
@@ -21,6 +22,63 @@ To update the go-ipfs, run:
 ```bash
 > make go.mod IPFS_VERSION=version
 ```
+## Detailed Installation
+For a brand new ipfs instance (no data stored yet)
+1. copy s3plugin.so $IPFS_DIR/plugins/go-ds-s3.so
```

Comment:
One should be able to run:

1. `ipfs init`.
2. `make install IPFS_VERSION=$VERSION`.

That will install the plugin into the IPFS repo (and fix permissions, etc.).

Reply:
Yep, but I'm not installing locally and thought this would be useful to know regardless. I can make the local option clearer

Reply:
the next thing I'm doing is to automate this whole process

Reply:
Awesome! Then look at https://github.com/ipfs/go-ds-s3/pull/25#issuecomment-521428605. We should be able to take an ipfs binary and build a matching plugin.

Reply:
Sorry I meant the install process, not the building. (to go with our current IPFS installer and runtime manager) - 
https://github.com/Peergos/Peergos/blob/master/src/peergos/server/storage/IpfsInstaller.java
https://github.com/Peergos/Peergos/blob/master/src/peergos/server/storage/IpfsWrapper.java